### PR TITLE
Update Source map to the latest changes in AST

### PR DESCRIPTION
**Pull request by zdne**
Remove the artificial example as its usability is rather low.

@pksunkara please treat is as non-authoritative proposal, if you have already implemented some parts of this in a different way, feel free to propose the change/update. Thanks.

### CODE REVIEW

File: Source Map.md [hunk 1]
```diff
@@ -24,17 +24,28 @@ Each block is defined by a zero-based index of its first character and the numbe
     - *1219* (number) - Zero-based index of the first character
     - *30* (number) - Number of the characters in the block
 
-### Blueprint Source Map (object)
+### Blueprint Source Map ([Element Source Map][])
```
Comment (pksunkara) [inline Source Map.md]: I don't think we want to add this. The AST doesn't contain something like this.

File: Source Map.md [hunk 2]
```diff
@@ -122,13 +133,8 @@ Source map of [Attributes][]
...
 
 #### Properties
-+ `source` ([Named Type Source Map][]) - Source map of the data structure as described in the source API Blueprint
-
-### Data Structures Source Map (object)
-Source map of [Data Structures][]
-
-#### Properties
-+ `types` (array[[Data Structure Source Map][]]) - Array of defined data structure source maps
++ `content`
```
Comment (pksunkara) [inline Source Map.md]:
```
### Data Structure Source Map ([Named Type Source Map][])
```

There is no content in here.

Comment (zdne) [inline Source Map.md]: @pksunkara are you suggesting to make `Data Structure Source Map` equal to `Named Type Source Map` ?

**Comment by zdne** [comment]
@pksunkara reverted blueprint source map base type

File: Source Map.md (hunk 2 above)
Comment (pksunkara) [inline Source Map.md]: No, don't change it. Forget my earlier comment.

**Pull request merged**